amun/entrypoint.py

```diff
@@ -119,5 +119,12 @@ def internal_server_error(exc):
     )
 
 
+@application.after_request
+def apply_headers(response):
+    """Add headers to each response."""
+    response.headers["X-Thoth-Version"] = __version__
```

Comment:
This is Amun service, the version string will differ from Thoth's version. You can change the name of the header to `X-Amun-Version`.

Reply:
Thanks, fixed!